test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -28,11 +29,9 @@
           "Microsoft.Azure.WebJobs.Script": "4.1033.6",
           "Microsoft.Azure.WebJobs.Script.Grpc": "4.1033.6",
           "Microsoft.Azure.WebSites.DataProtection": "2.1.91-alpha",
-          "Microsoft.Identity.Client": "4.60.3",
```

Comment:
This is private, but is this related to your change? I wouldn't expect it to be the case.

Did it cause a test failure flagging the removal? If not, can you please revert this change (and the SourceLink one) so we can clean this in a separate PR?

Reply:
It's not related to my change and indeed doesn't cause the deps test to fail. Just reverted them and let the addition of Azure.Data.Tables.